src/components/lyne-link/styles/lyne-link.component.scss

```diff
@@ -0,0 +1,70 @@
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin, available for shared
```

Comment:
I did not look into it any further, but the storybook plugin for switching direction ltr <> rtl is not working. But the semantic of your component is same as with the button, and there it is working (flex element with 2 spans in it). Did you notice it too? any idea how we can fix it?